cmd_clean/main.go

```diff
@@ -0,0 +1,65 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/clean"
+	"github.com/vmware-tanzu/nsx-operator/pkg/config"
+	"github.com/vmware-tanzu/nsx-operator/pkg/logger"
+)
+
+// usage:
+// ./bin/clean -cluster='domain-c9:47b14f07-33b4-478b-8025-a6ae16e81be7'  -thumbprint="0E:CA:B5:98:D4:46:8C:19:4A:54:4A:67:7C:7B:9D:22:0F:78:76:5E" -log-level=0 -vc-user="administrator@vsphere.local" -vc-passwd="RAZKssvew+FyR7+7" -vc-endpoint="sc2-10-187-150-214.nimbus.eng.vmware.com" -vc-sso-domain="vsphere.local" -vc-https-port=443  -mgr-ip="10.187.157.154"
+
+var (
+	log          = logger.Log
+	cf           *config.NSXOperatorConfig
+	mgrIp        string
+	vcEndpoint   string
+	vcUser       string
+	vcPasswd     string
+	vcSsoDomain  string
+	vcHttpsPort  int
+	thumbprint   string
```

Comment:
Could we also support nsx ca cert as input, we will remove SHA1 thumbprint support soon

Reply:
fixed.